### PR TITLE
Unify Backup Modals

### PR DESCRIPTION
**Pull request by Jon-edge**
Could consolidate a bit further, but avoiding extra work until after experiment is concluded.

"Original" - Settings, deleting light accounts, receiving FIO requests
image

"Original emergency hack" / "Control Variant" - Our quick and dirty message after realizing people could lose funds. Refreshed to be generally consistent with other backup modals
image

Other Variants:
image
image
image

### CHANGELOG

Does this branch warrant an entry to the CHANGELOG?

- [x] Yes
- [ ] No

### Dependencies

https://github.com/EdgeApp/edge-react-gui/pull/5009

### Requirements

If you have made **any** visual changes to the GUI. Make sure you have:

- [x] Tested on iOS device
- [x] Tested on Android device
- [ ] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207024313887978

### CODE REVIEW

File: src/components/modals/ButtonsModal.tsx [hunk 1]
```diff
@@ -62,42 +61,39 @@ export function ButtonsModal<Buttons extends { [key: string]: ButtonInfo }>(prop
...
-            }
-
-            return <ButtonUi4 key={key} label={label} marginRem={0.5} type={type} onPress={handlePress} layout="column" />
-          })}
-        </View>
+        <ModalButtons
+          primary={buttonInfo.length > 0 ? buttonInfo[0] : undefined}
+          secondary={buttonInfo.length > 1 ? buttonInfo[1] : undefined}
+          tertiary={buttonInfo.length > 2 ? buttonInfo[2] : undefined}
+        />
```
Comment (swansontec) [inline src/components/modals/ButtonsModal.tsx:96]: This is a pretty big change! All the buttons modals in the app are going to have their buttons types affected, which will be a big QA effort. This should be its own PR, and in that PR, we'll have to audit all the places where we use this, and we will have to remove the `theme.preferPrimaryButton`.

**Review by swansontec — CHANGES_REQUESTED** [review]
The backup modal changes look OK, but the changes to ButtonsModal are too broad-reaching to slip into this PR. Please skip those out-of-scope commits, and submit them separately.

It should be pretty easy to add `type="primary"` to the `BackupModal` buttons, so I don't think this shoud be a difficult rebase.

File: src/components/modals/ButtonsModal.tsx (hunk 1 above)
Comment (Jon-edge) [reply src/components/modals/ButtonsModal.tsx:96]: Yeah this bigger change came from when I resolved with Madison that this is the way we are going regarding modal buttons. I confirmed with Paul if it was safe for white labels to deprecate `theme.preferPrimaryButton` and this is the last remaining place we use `theme.preferPrimaryButton`.

Comment (Jon-edge) [reply src/components/modals/ButtonsModal.tsx:96]: Created https://github.com/EdgeApp/edge-react-gui/pull/5009

**Review by Jon-edge — COMMENTED** [review]
Created https://github.com/EdgeApp/edge-react-gui/pull/5009

**Review by swansontec — APPROVED** [review]
Thanks for getting this fixed.
